lib/worksheet.js

```diff
@@ -76,6 +75,14 @@ function XlsxStreamReaderWorkSheet (workBook, sheetName, workSheetId, workSheetS
 }
 Util.inherits(XlsxStreamReaderWorkSheet, Stream)
 
+const tryFormat = function (format, workingVal) {
```

Comment:
I tried to refactor a bit,  complexity is beginning to pile up in the formatting code